insights/parsers/ethtool.py

```diff
@@ -501,9 +501,10 @@ def parse_content(self, content):
         for line in content[2:]:
             if line.strip():
                 (key, value) = [s.strip() for s in line.split(":", 1)]
-                value = int(value)
-                self.data[key] = value
-                setattr(self, key.replace("-", "_"), value)
+                if 'n/a' not in value:
```

Comment:
@chenlizhong Looking at the exceptions that you posted, would it be better to check the type of the value variable and make sure it's capable to be passed to int? Like the below. That way if another case of the variable ends up being a string it will be skipped as well.

```
if value.isnumeric():
    value = int(value)
    self.data[key] = value
    setattr(self, key.replace("-", "_"), value)
```

Reply:
Thanks. Updated the code.

Reply:
Awesome thank you it looks good to me, but there is one failed test due to python 2.6's sys.version_info not having named variables.